src/compiler/types.ts

```diff
@@ -5666,7 +5680,26 @@ export enum TypeReferenceSerializationKind {
 }
 
 /** @internal */
-export interface EmitResolver {
+export interface CoreEmitResolver {

```

Comment:
hmm, I wonder why there are new Core* versions of interfaces. Is it Core=public, original=internal? But the new ones are marked internal too.